scripts/check-migrations.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+from shutil import ExecError
+
+"""
+This script is meant to be run in CI to check that migrations changes are
+not coupled with other changes.
+"""
+
+
+class CoupledMigrations(Exception):
+    pass
+
+
+def _get_migration_changes(workdir: str = ".", to: str = "origin/master") -> str:
+    return _get_changes("snuba/snuba_migrations/*/[0-9]*.py", workdir, to)
```

Comment:
This needs to also allow changing the migration group to add the migration (https://github.com/getsentry/snuba/blob/master/snuba/migrations/groups.py).

You might also want to allow test files.

Reply:
fixed. I updated it to use an `ALLOWED_MIGRATIONS` list that ignores tests, docs and groups.py in the git diff.